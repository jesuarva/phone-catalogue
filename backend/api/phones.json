[{
    "Model": "Razer Phone",
    "CPU": "Snapdragon 835",
    "CPU Specifications": "Octa-core, \n\n2.35 GHz",
    "GPU": "Adreno 540",
    "Storage capacity": "64 GB",
    "Removable storage": "256GB microSDXC",
    "RAM": "8 GB",
    "OS": "Android 7.1 \"Nougat",
    "Custom Launcher": "",
    "Dimensions": "158.5 x 77.7 x \n\n8 mm",
    "Weight": "197 g",
    "Battery": "4000 mAh Li-Ion, non-removable battery",
    "Display": "5.7\", 2560x1440, 120 Hz",
    "Camera": "Dual 12 MP (rear camera), 8 MP (front camera)",
    "image": "https://cdn2.gsmarena.com/vv/bigpic/razer-phone.jpg"
  },
  {
    "Model": "Samsung Galaxy Note 8",
    "CPU": "Snapdragon 835 \n\nExynos 8895",
    "CPU Specifications": "Octa-core, \n\n2.35 GHz",
    "GPU": "Adreno 540 \n\nARM Mali-G71 MP20",
    "Storage capacity": "64/128/256 GB",
    "Removable storage": "256GB microSDXC",
    "RAM": "6 GB",
    "OS": "Android 7.1.1 \"Nougat",
    "Custom Launcher": "Samsung Experience",
    "Dimensions": "162.5 x 74.8 x \n\n8.6 mm",
    "Weight": "195 g",
    "Battery": "3300 mAh Li-Ion, non-removable battery",
    "Display": "6.3\", 2960x1440, AMOLED",
    "Camera": "Dual 12 MP (rear camera), 8 MP (front camera)",
    "image": "https://s7d2.scene7.com/is/image/SamsungUS/Note8-Front-S-Pen-Orchidgray?$product-details-jpg$"
  },
  {
    "Model": "Essential Phone",
    "CPU": "Snapdragon 835",
    "CPU Specifications": "Octa-core, \n\n2.45 GHz",
    "GPU": "Adreno 540",
    "Storage capacity": "128 GB",
    "Removable storage": "None",
    "RAM": "4 GB",
    "OS": "Android 7.1.1 \"Nougat",
    "Custom Launcher": "",
    "Dimensions": "141.5 x 71.1 x \n\n7.8 mm",
    "Weight": "185 g",
    "Battery": "3,040 mAh",
    "Display": "5.71\", 2560 x 1312",
    "Camera": "13 MP color + 13 MP monochrome \n\n(rear camera),\n\n8 MP (front camera)",
    "image": "https://images-na.ssl-images-amazon.com/images/I/81OvOMun5zL._SX425_.jpg"
  },
  {
    "Model": "Pixel 2 XL",
    "CPU": "Snapdragon 835",
    "CPU Specifications": "Octa-core, \n\n2.45 GHz",
    "GPU": "Adreno 540",
    "Storage capacity": "64 GB or 128 GB",
    "Removable storage": "None",
    "RAM": "4 GB LPDDR4X",
    "OS": "Android 8.0 \"Oreo",
    "Custom Launcher": "",
    "Dimensions": "157.9 x 76.7 x \n\n7.9 mm",
    "Weight": "175 g",
    "Battery": "3,520 mAh",
    "Display": "6\", 2880 × 1440, P-OLED",
    "Camera": "12.2 MP (rear camera), 8 MP (front camera)",
    "image": "https://images-na.ssl-images-amazon.com/images/I/71o2AW-OlFL._SX569_.jpg"
  },
  {
    "Model": "Pixel 2",
    "CPU": "Snapdragon 835",
    "CPU Specifications": "Octa-core, \n\n2.45 GHz",
    "GPU": "Adreno 540",
    "Storage capacity": "64 GB or 128 GB",
    "Removable storage": "None",
    "RAM": "4 GB LPDDR4X",
    "OS": "Android 8.0 \"Oreo",
    "Custom Launcher": "",
    "Dimensions": "145.7 x \n\n69.7 x 7.8 mm",
    "Weight": "143 g",
    "Battery": "2,700 mAh",
    "Display": "5\", 1920 × 1080, AMOLED",
    "Camera": "12.2 MP (rear camera), 8 MP (front camera)",
    "image": "https://images-na.ssl-images-amazon.com/images/I/411ZLDLOFTL.jpg"
  },
  {
    "Model": "Huawei P10",
    "CPU": "HiSilicon Kirin 960",
    "CPU Specifications": "Octa-core, 2.4 GHz",
    "GPU": "Mali-G71 MP8",
    "Storage capacity": "64 GB or 128 GB",
    "Removable storage": "256GB MicroSD",
    "RAM": "4 GB (P10), \n\n6 GB (P10 Plus)",
    "OS": "Android 7.0 Nougat",
    "Custom Launcher": "Emotion UI 5.1",
    "Dimensions": "145.3 x 69.3 x \n\n6.98 mm (P10)\n\n153.5 x 74.2 x\n\n6.98 mm\n\n(P10 Plus)",
    "Weight": "145 g (P10) \n\n165 g (P10 Plus)",
    "Battery": "3200 mAh(P10), \n\n3750 mAh(P10 Plus)",
    "Display": "5.1\" 1920×1080 (P10), 5.5\" 2560×1440 (P10 Plus)",
    "Camera": "12 MP and 20 MP (rear) \n\n8 MP (front camera)",
    "image": "https://drop.ndtv.com/TECH/product_database/images/226201773632PM_635_huawei_p10.jpeg"
  },
  {
    "Model": "Huawei Mate 10 Pro",
    "CPU": "HiSilicon Kirin 970",
    "CPU Specifications": "Octa-core, 4x2.4 GHz Cortex-A73 & 4x1.8 GHz Cortex-A53",
    "GPU": "Mali-G72 MP12",
    "Storage capacity": "64 GB or 128 GB",
    "Removable storage": "No",
    "RAM": "4 GB (64 GB), \n\n6 GB (128 GB)",
    "OS": "Android 8.0 Oreo",
    "Custom Launcher": "Emotion UI 5.1",
    "Dimensions": "154.2 x 74.5 x \n\n7.9 mm",
    "Weight": "178 g",
    "Battery": "4000 mAh,",
    "Display": "6.0\" 1080×2160, 18:9 ratio",
    "Camera": "Rear (Dual) : 12 MP (f/1.6, OIS) + 20 MP (f/1.6) \n\nFront (Single): 8 MP; (f/2.0)",
    "image": "https://www.senheng.com.my/media/catalog/product/cache/3/thumbnail/600x/17f82f742ffe127f42dca9de82fb58b1/m/a/mate10pro-blue.jpg"
  },
  {
    "Model": "HTC U11",
    "CPU": "Snapdragon 835",
    "CPU Specifications": "Octa-core, \n\n2.45 GHz",
    "GPU": "Adreno 540",
    "Storage capacity": "64 GB or 128 GB",
    "Removable storage": "256GB MicroSD",
    "RAM": "4 or 6 GB",
    "OS": "Android 7.1 \"Nougat",
    "Custom Launcher": "HTC Sense 8.0",
    "Dimensions": "153.9 x 75.9 x \n\n7.9 mm",
    "Weight": "169 g",
    "Battery": "3000 mAh",
    "Display": "5.5\" 2560×1440",
    "Camera": "12 MP (rear) \n\n16 MP (front camera)",
    "image": "https://media.wired.com/photos/5a6671b83e45fc27d573835c/master/w_1258,c_limit/HTC-u11-life-TopArt.jpg"
  },
  {
    "Model": "Nokia 8",
    "CPU": "Snapdragon 835",
    "CPU Specifications": "Octa-core, \n\n2.45 GHz",
    "GPU": "Adreno 540",
    "Storage capacity": "64 GB or 128 GB",
    "Removable storage": "256GB MicroSD",
    "RAM": "4 or 6 GB",
    "OS": "Android 7.1 \"Nougat",
    "Custom Launcher": "",
    "Dimensions": "151.5 × 73.7 \n\n× 7.9 mm",
    "Weight": "160 g",
    "Battery": "3090 mAh Li-Po, non removable battery",
    "Display": "5.3\" 2560 x 1440",
    "Camera": "13 MP (rear) \n\n13 MP (front camera)",
    "image": "https://cdn.eglobalcentral.eu/images/detailed/60/nokia-8-dual-sim-4gb-ram-64gb-glossy-copper.jpg"
  },
  {
    "Model": "LG G6",
    "CPU": "Snapdragon 821",
    "CPU Specifications": "Dual-core 2.35 GHz & Dual-core 1.6 GHz Kryo",
    "GPU": "Adreno 530",
    "Storage capacity": "64 GB or 32 GB",
    "Removable storage": "2000 GB (2TB max.) microSDXC",
    "RAM": "4 GB LPDDR4 RAM",
    "OS": "Android 7.0 \"Nougat",
    "Custom Launcher": "LG UX",
    "Dimensions": "148.9 x 71.9 x 7.9 mm",
    "Weight": "162 g",
    "Battery": "3300 mAh Li-Po, non removable battery",
    "Display": "5.7\", 2880×1440",
    "Camera": "Dual 13 MP (rear camera), 5 MP (front camera)",
    "image": "https://d2giyh01gjb6fi.cloudfront.net/default/0001/60/thumb_59655_default_big.jpeg"
  },
  {
    "Model": "LG V30",
    "CPU": "Snapdragon 835",
    "CPU Specifications": "Octa-core (4x2.45 GHz & 4x1.9 GHz) Kryo",
    "GPU": "Adreno 540",
    "Storage capacity": "64 GB (V30) or 128 GB (V30+)",
    "Removable storage": "Up to 2 TB MicroSD",
    "RAM": "4 GB",
    "OS": "Android 7.1.2 \"Nougat",
    "Custom Launcher": "",
    "Dimensions": "151.7 mm x \n\n75.4 mm x\n\n7.4 mm",
    "Weight": "158 g",
    "Battery": "3300 mAh Li-Po, non removable battery",
    "Display": "6\", 2880×1440, P-OLED",
    "Camera": "16 MP & 13 MP wide-angle (rear), 5 MP (front)",
    "image": "https://cdn.movertix.com/media/catalog/product/cache/image/600x/l/g/lg-v30-en-azul-de-64gb-y-4gb-ram-h930.jpg"
  },
  {
    "Model": "Huawei Honor 9",
    "CPU": "HiSilicon Kirin 960",
    "CPU Specifications": "Octa-core, 2.4 GHz",
    "GPU": "Mali-G71 MP8",
    "Storage capacity": "64 GB or 128 GB",
    "Removable storage": "256GB MicroSD",
    "RAM": "4 or 6 GB",
    "OS": "Android 7.0 \"Nougat",
    "Custom Launcher": "Emotion UI 5.1",
    "Dimensions": "147 x \n\n70.9 x\n\n7.5 mm",
    "Weight": "159 g",
    "Battery": "3200 mAh",
    "Display": "5.15\", 1920 × 1080",
    "Camera": "Dual 20 MP (rear camera), 8 MP (front camera)",
    "image": "https://cdn.shopify.com/s/files/1/1374/6193/products/000693-huawei-honor-9-glacier-grey-3.jpg?v=1503007820"
  },
  {
    "Model": "BlackBerry KEYone",
    "CPU": "Snapdragon 625",
    "CPU Specifications": "Octa-Core 2.0 GHz - Cortex-A53",
    "GPU": "Adreno 506",
    "Storage capacity": "32 GB",
    "Removable storage": "2000 GB (2TB max.) microSDXC",
    "RAM": "3 GB LPDDR3",
    "OS": "Android 7.1.1 \"Nougat",
    "Custom Launcher": "",
    "Dimensions": "149.3 x 72.5 x 9.4 mm",
    "Weight": "180 g",
    "Battery": "3505 mAh Li-Ion non-removable battery",
    "Display": "4.5\", 1620×1080",
    "Camera": "12 MP (rear camera), \n\n8 MP (front camera)",
    "image": "https://gloimg.gbtcdn.com/gb/pdm-product-pic/Electronic/2017/09/21/source-img/20170921102421_69887.jpg"
  },
  {
    "Model": "Sony Xperia XZ Premium",
    "CPU": "Snapdragon 835",
    "CPU Specifications": "Quad-core 2.45 GHz Kryo 280 + Quad-core 1.9 GHz Kryo 280",
    "GPU": "Adreno 540",
    "Storage capacity": "64GB",
    "Removable storage": "Up to 256GB",
    "RAM": "4 GB",
    "OS": "Android 7.1 \"Nougat",
    "Custom Launcher": "",
    "Dimensions": "156 x 77 x 7.9 mm",
    "Weight": "195 g",
    "Battery": "3230 mAh Li-ion non-removable battery",
    "Display": "5.46\", 3840×2160",
    "Camera": "19 MP (rear camera), \n\n13 MP (front camera)",
    "image": "https://api.sonymobile.com/files/xperia-XZ-Premium-Silver-product-shot-2000x2000-eb3b7f375688f9b33780aed533d4f6eb.png"
  },
  {
    "Model": "Samsung Galaxy S8",
    "CPU": "Snapdragon 835 \n\nExynos 8895[1]",
    "CPU Specifications": "Octa-core 2.35 GHz",
    "GPU": "Adreno 540 \n\nARM Mali-G71 MP20",
    "Storage capacity": "64/128GB",
    "Removable storage": "256GB microSDXC",
    "RAM": "4 GB",
    "OS": "Android 7.0 \"Nougat",
    "Custom Launcher": "Samsung Experience",
    "Dimensions": "148.9 x 68.1 x 8.0 mm",
    "Weight": "152g",
    "Battery": "3000 mAh Li-Ion non-removable battery",
    "Display": "5.8\", 2960×1440, AMOLED",
    "Camera": "12 MP (rear camera), \n\n8 MP (front camera)",
    "image": "https://s7d2.scene7.com/is/image/SamsungUS/S8?$configurator-model-view-jpg$"
  },
  {
    "Model": "Apple iPhone 8 Plus",
    "CPU": "Apple A11 Bionic",
    "CPU Specifications": "Hexa-core (2× Monsoon + 4× Mistral) 2.39 GHz",
    "GPU": "Apple designed three-core GPU",
    "Storage capacity": "64 GB or 256 GB",
    "Removable storage": "no external memory slot",
    "RAM": "3 GB LPDDR4X",
    "OS": "iOS 11",
    "Custom Launcher": "N/A",
    "Dimensions": "158.4 x 78.1 x 7.5 mm",
    "Weight": "202g",
    "Battery": "2675 mAh Li-Po non-removable battery",
    "Display": "5.5\", 1920x1080",
    "Camera": "12 MP (rear camera), \n\n7 MP (front camera)",
    "image": "https://store.storeimages.cdn-apple.com/4981/as-images.apple.com/is/image/AppleInc/aos/published/images/i/ph/iphone8/select/iphone8-select-2017?wid=212&hei=292&fmt=png-alpha&.v=1503618523038"
  },
  {
    "Model": "Apple iPhone X",
    "CPU": "Apple A11 Bionic",
    "CPU Specifications": "Hexa-core (2× Monsoon + 4× Mistral) 2.39 GHz",
    "GPU": "Apple designed three-core GPU",
    "Storage capacity": "64 GB or 256 GB",
    "Removable storage": "no external memory slot",
    "RAM": "3 GB LPDDR4X",
    "OS": "iOS 11",
    "Custom Launcher": "N/A",
    "Dimensions": "143.6 x 70.9 x 7.7 mm",
    "Weight": "174g",
    "Battery": "2716 mAh Li-Po non-removable battery",
    "Display": "5.8\", 2436x1125, OLED",
    "Camera": "12 MP (rear camera), \n\n7 MP (front camera)",
    "image": "https://brain-images-ssl.cdn.dixons.com/4/8/10168684/u_10168684.jpg"
  }
]