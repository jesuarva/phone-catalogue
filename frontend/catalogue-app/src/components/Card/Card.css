.custom-card {
  min-height: 30vh;
}

.custom-card a h1 {
  text-decoration: none !important;
}