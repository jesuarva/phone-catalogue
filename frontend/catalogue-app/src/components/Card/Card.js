import React from 'react';
import { Link } from 'react-router-dom';
import { Jumbotron } from 'reactstrap';
import Pic from '../Pic/Pic';
import './Card.css';

const Card = ({ phone }) => {
  const { image, Model } = phone;
  const urlModel = Model.replace(/\s/g, '-');
  return (
    <div className="col-12 col-md-4 custom-card">
      <Link to={`/catalogue/${urlModel}`}>
        <Jumbotron className="custom-card">
          <Pic img={image} altAttribute={Model} />
          <h1 className="display-6">{Model}</h1>
        </Jumbotron>
      </Link>
    </div>
  );
};

export default Card;
