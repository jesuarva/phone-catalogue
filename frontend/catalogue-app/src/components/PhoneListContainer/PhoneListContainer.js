import React, { Fragment } from 'react';
import { connect } from 'react-redux';
import Card from '../Card/Card';

const PhoneListContainer = (props) => {
  const { data } = props;
  return (
    <div className="row">
      {data.map((phone, i) => (
        <Card
          key={Math.random() + Date.now()}
          id={`phone-${i}`}
          phone={phone}
        />
      ))}
    </div>
  );
};

const mapStateToProps = (state) => {
  return {
    data: state.data,
  };
};

export default connect(
  mapStateToProps,
  {},
)(PhoneListContainer);
