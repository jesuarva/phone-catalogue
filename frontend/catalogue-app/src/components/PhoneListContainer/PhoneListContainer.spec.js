import { configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import phones from '../../../../../backend/api/phones.json';
import Card from '../Card/Card';
import TestWrapper from '../TestWrapper/TestWrapper';
import { PhoneListContainer } from './PhoneListContainer.js';

configure({ adapter: new Adapter() });

describe('PhoneListContainer component', () => {
  function setUp() {
    const props = {
      data: phones,
    };

    const component = (
      <Router>
        <PhoneListContainer {...props} />
      </Router>
    );

    return {
      props,
      component,
    };
  }

  it('should renders correctly', () => {
    const { component } = setUp();
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should renders all Card coponents', () => {
    const { component } = setUp();
    const mounted = mount(<TestWrapper>{() => component}</TestWrapper>);
    const cards = mounted.find(Card);
    expect(cards).toHaveLength(17);
  });
});
