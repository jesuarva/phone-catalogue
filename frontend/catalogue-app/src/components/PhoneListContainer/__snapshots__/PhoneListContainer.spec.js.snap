// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhoneListContainer component should renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Razer-Phone"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Razer Phone"
              className="figure-img img-fluid rounded"
              src="https://cdn2.gsmarena.com/vv/bigpic/razer-phone.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Razer Phone
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Samsung-Galaxy-Note-8"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Samsung Galaxy Note 8"
              className="figure-img img-fluid rounded"
              src="https://s7d2.scene7.com/is/image/SamsungUS/Note8-Front-S-Pen-Orchidgray?$product-details-jpg$"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Samsung Galaxy Note 8
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Essential-Phone"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Essential Phone"
              className="figure-img img-fluid rounded"
              src="https://images-na.ssl-images-amazon.com/images/I/81OvOMun5zL._SX425_.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Essential Phone
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Pixel-2-XL"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Pixel 2 XL"
              className="figure-img img-fluid rounded"
              src="https://images-na.ssl-images-amazon.com/images/I/71o2AW-OlFL._SX569_.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Pixel 2 XL
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Pixel-2"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Pixel 2"
              className="figure-img img-fluid rounded"
              src="https://images-na.ssl-images-amazon.com/images/I/411ZLDLOFTL.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Pixel 2
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Huawei-P10"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Huawei P10"
              className="figure-img img-fluid rounded"
              src="https://drop.ndtv.com/TECH/product_database/images/226201773632PM_635_huawei_p10.jpeg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Huawei P10
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Huawei-Mate-10-Pro"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Huawei Mate 10 Pro"
              className="figure-img img-fluid rounded"
              src="https://www.senheng.com.my/media/catalog/product/cache/3/thumbnail/600x/17f82f742ffe127f42dca9de82fb58b1/m/a/mate10pro-blue.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Huawei Mate 10 Pro
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/HTC-U11"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="HTC U11"
              className="figure-img img-fluid rounded"
              src="https://media.wired.com/photos/5a6671b83e45fc27d573835c/master/w_1258,c_limit/HTC-u11-life-TopArt.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          HTC U11
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Nokia-8"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Nokia 8"
              className="figure-img img-fluid rounded"
              src="https://cdn.eglobalcentral.eu/images/detailed/60/nokia-8-dual-sim-4gb-ram-64gb-glossy-copper.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Nokia 8
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/LG-G6"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="LG G6"
              className="figure-img img-fluid rounded"
              src="https://d2giyh01gjb6fi.cloudfront.net/default/0001/60/thumb_59655_default_big.jpeg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          LG G6
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/LG-V30"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="LG V30"
              className="figure-img img-fluid rounded"
              src="https://cdn.movertix.com/media/catalog/product/cache/image/600x/l/g/lg-v30-en-azul-de-64gb-y-4gb-ram-h930.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          LG V30
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Huawei-Honor-9"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Huawei Honor 9"
              className="figure-img img-fluid rounded"
              src="https://cdn.shopify.com/s/files/1/1374/6193/products/000693-huawei-honor-9-glacier-grey-3.jpg?v=1503007820"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Huawei Honor 9
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/BlackBerry-KEYone"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="BlackBerry KEYone"
              className="figure-img img-fluid rounded"
              src="https://gloimg.gbtcdn.com/gb/pdm-product-pic/Electronic/2017/09/21/source-img/20170921102421_69887.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          BlackBerry KEYone
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Sony-Xperia-XZ-Premium"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Sony Xperia XZ Premium"
              className="figure-img img-fluid rounded"
              src="https://api.sonymobile.com/files/xperia-XZ-Premium-Silver-product-shot-2000x2000-eb3b7f375688f9b33780aed533d4f6eb.png"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Sony Xperia XZ Premium
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Samsung-Galaxy-S8"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Samsung Galaxy S8"
              className="figure-img img-fluid rounded"
              src="https://s7d2.scene7.com/is/image/SamsungUS/S8?$configurator-model-view-jpg$"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Samsung Galaxy S8
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Apple-iPhone-8-Plus"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Apple iPhone 8 Plus"
              className="figure-img img-fluid rounded"
              src="https://store.storeimages.cdn-apple.com/4981/as-images.apple.com/is/image/AppleInc/aos/published/images/i/ph/iphone8/select/iphone8-select-2017?wid=212&hei=292&fmt=png-alpha&.v=1503618523038"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Apple iPhone 8 Plus
        </h1>
      </div>
    </a>
  </div>
  <div
    className="col-12 col-md-4 custom-card"
  >
    <a
      href="/catalogue/Apple-iPhone-X"
      onClick={[Function]}
    >
      <div
        className="custom-card jumbotron"
      >
        <div
          className="pic"
        >
          <figure
            className="figure"
          >
            <img
              alt="Apple iPhone X"
              className="figure-img img-fluid rounded"
              src="https://brain-images-ssl.cdn.dixons.com/4/8/10168684/u_10168684.jpg"
            />
          </figure>
        </div>
        <h1
          className="display-6"
        >
          Apple iPhone X
        </h1>
      </div>
    </a>
  </div>
</div>
`;
